lib/commands/exam.js

```diff
@@ -3,8 +3,12 @@
 var fs = require('fs-extra');
 var path = require('path');
 var TestCommand = require('ember-cli/lib/commands/test');
+var testServerTask = require('./task/test-server');
+var testTask = require('./task/test');
 var debug = require('debug')('exam');
 
+const executionMappingObj = {};
```

Comment:
Is this intended to be used as a dictionary? If so, you may want to consider using `Object.create(null)` so it doesn't inherit.